### ISSUE TITLE
Bucket Interaction

### ISSUE THREAD

**Issue by ZeekDaGeek** [Drayshak/EnderTech#35]
SUGGESTION / FEATURE REQUEST

It would be nice to be able to fill and empty buckets by right clicking on the tanks similar to how you do with many other tanks. Additional right clicking to bring up the interface and having a slot to fill and empty containers would be very nice for things that don't have right click interactions. Each action could take 1,000 RF to fit the rest of the balance.

**Comment by Drayshak** [comment]
Yeah I like that idea, I'll take a look in the next few days. It would probably be limited to right clicking on valves, as that's the only point of IO that there is right now.
